developer-docs-site/docs/concepts/txns-states.md

```diff
@@ -106,25 +98,23 @@ All operations executed by the blockchain and all account states can be verified
 - The validator nodes agree on the state. 
 - The client does not need to trust the entity from which it is receiving data. For example, if a client fetches the last **n** transactions from an account, a proof can attest that no transactions were added, omitted or modified in the response. The client may also query for the state of an account, ask whether a specific transaction was processed, and so on.
 
-## Ledger state
```

Comment:
Why delete this?

Reply:
it wasn't, I moved it to "## State" and moved it to before proofs. look up a few lines